modules/system/chat-wfrp4e.js

```diff
@@ -414,7 +418,7 @@ export default class ChatWFRP {
     let message = game.messages.get(msgId)
     let conditionResult;
 
-    if (combatant.actor.isOwner)
+    if (combatant.actor.isOwner && game.user.isGM)
```

Comment:
Why was this added? Players applying their own conditions is preferable

Reply:
crap, sorry, thanks for spotting this, it was for my personal use.